test/SCRStub.rb

```diff
@@ -58,3 +58,25 @@ def stub_scr_read(path_name)
     allow(Yast::SCR).to receive(:Read).with(path).and_return info
   end
 end
+
+module YaPINetworkStub
```

Comment:
I change it from NetworkInterfacesStub to YaPINetworkStub, so it contains some methods to stub models called by YaPINetwork.

Yeah, it should better to replace it.